lib/emis/service.rb

```diff
@@ -19,29 +19,37 @@ class Service < Common::Client::Base
     #
     # @param endpoints [Array<String, Symbol, Array<String, Symbol>>] An array of endpoints,
     #  either a string or symbol if the method name and endpoint path (converted to camelcase)
-    #  are the same or an Array containing the method name and endpoint path
+    #  are the same or an Array containing the method name, endpoint path,
+    #  and optional version of response and soapaction
+    #
+    # rubocop:disable Metrics/MethodLength
     def self.create_endpoints(endpoints)
       endpoints.each do |endpoint|
         operation = nil
         request_name = nil
         if endpoint.is_a?(Array)
```

Comment:
We can probably extract this if/else to a separate method to satisfy rubocop instead of disabling the metric. It has clear inputs and outputs.